### DISCUSSION TITLE
Unable to load entities

### DISCUSSION THREAD

**Discussion by LukeMc8** [nathanmarlor/foxess_modbus#380] [Support]
Hi, 
    I'm having a bit of trouble getting this card to work, after installing and choosing my invertor from the drop down all i get is a heading stating unable to load the following sensors and then a list of the sensors. My integration is fully upto date and i have down a number of reboots but it doesnt seem to work for me. 

Do you have any ideas?

Luke

**Comment by canton7** [comment]
Can you post the exact error message, including the entity IDs it couldn't load? What are the entity IDs of e.g. your "Period 1 - Enable Charge from Grid", "Period 1 - Enable Force Charge", "Period 1 - Start" and "Period 1 - End" entities?

**Comment by LukeMc8** [comment]
Hi, hopefully this screenshot gives you the info?

![Screenshot](https://github.com/nathanmarlor/foxess_modbus/assets/8438238/ba423060-a9d4-4b0f-b715-f6535650fa5f)

**Reply by canton7** [reply]
Thanks. Can you provide the other information I asked for as well, please?

**Comment by LukeMc8** [comment]
Period 1 enable charge from grid = binary_sensor.fox_h1_time_period_1_enable_charge_from_grid
Period 1 enable force charge = binary_sensor.fox_h1_time_period_1_enable_force_charge
Period 1 start = sensor.fox_h1_time_period_1_start
Period 1 end = sensor.fox_h1_time_period_1_end

Do you need anything else?

**Reply by canton7** [reply]
Thanks. Looks like you set up the integration at a point in time when we didn't properly enforce that the friendly name you chose was valid in an entity ID. I'll get a workaround into foxess_modbus

**Comment by LukeMc8** [comment]
Thanks. Do you think i will continue to have issues like this in the future, is it worth me thinking about removing and re adding? Not sure how much of a pain that will be mind.

**Reply by canton7** [reply]
I'm going to put a proper fix into foxess_modbus to change your entity id prefix, so hopefully this shouldn't pop up again in the future!

That said, removing/readding is painless. Check the checkbox to supply your own entity id prefix, then specify "fox_h1".

**Comment by LukeMc8** [comment]
I've just re added and its working now. Thanks for the help and the excelent integrations.
